resources/views/cruds/fields/location.blade.php

```diff
@@ -12,7 +12,7 @@
 @endphp
 <x-forms.foreign
     :campaign="$campaign"
-    :name="isset($isParent) ? 'parent_location_id' : 'location_id'"
+    :name='"location_id"'
```

Comment:
Confused by the quotes? If passing a string, just `name="location_id"` is enough, `:` is for a php property.